mne_bids/utils.py

```diff
@@ -19,12 +19,27 @@
 from mne import read_events, find_events
 from mne.channels import read_montage
 from mne.io.pick import pick_types
+from mne.utils import check_version
 
 from .config import BIDS_VERSION
 from .io import _parse_ext
 from .tsv_handler import _to_tsv, _tsv_to_str
 
 
+def assert_mne_version(min_version):
```

Comment:
we don't need a separate public function for this. Just use `check_version` directly

Reply:
sure, for just solving my issue it'd be less code. 

but to be honest: i dont think it'd be a one-time thing to use. i am just peaking into mne-bids for two months and already reported two issues due to mne-version changes (+ with just one data format)

an assert wrapper is easier to interpret in the code, and allows to give the user a consistent error message

Reply:
well, as of now we have only one function. So you just need one call to `check_version`. Maybe the right way to solve this issue is to use a requirements.txt

Reply:
I agree that using `check_version` from MNE is probably sufficient. Can you clarify what you mean by using a `requirements.txt` @jasmainak ?

Reply:
I mean, I don't get how people can have the wrong version if they followed the installation instructions. We can't put a `check_version` in every function ... it's not how it's done in any library.

Reply:
wow, how nice :)

well, for sure some people will have version mismatchs. at least if `mne` will be adapted by a broader set of end-users and applied in projects that go over months or years.

anyway, i agree: one assertion in the beginning of `write_raw_bids` is a practical solution.